src/agent/rustjail/src/container.rs

```diff
@@ -838,8 +838,8 @@ impl BaseContainer for LinuxContainer {
 
         //restore the parent's process's pid namespace.
         defer!({
-            sched::setns(old_pid_ns, CloneFlags::CLONE_NEWPID);
-            unistd::close(old_pid_ns);
+            let _setns_ok = sched::setns(old_pid_ns, CloneFlags::CLONE_NEWPID);
+            let _ns_closed = unistd::close(old_pid_ns);
```

Comment:
Likewise can use `let _ = ` here.

Reply:
I agree with @dgibson, the meaningful names for unused variables are meaningless.